audit.js

```diff
@@ -1,5 +1,11 @@
 'use strict';
 
+/**
+ * Lists runs which do not have any of the three 'base' labels - master,
+ * pr_base, or pr_head. Such runs should be categorized into one of those three
```

Comment:
Some runs truly don't belong in any of the three buckets, namely full runs triggered from branches other than master.

Reply:
Oh, I wonder if those are the 2019 runs I've been picking up. Perhaps runs from pushes to the `trigger/*` branches...